vc-verifier/kotlin/README.md

```
@@ -830,6 +905,30 @@ For other unknown exceptions, error code will be `ERR_GENERIC`
 | iat in KB JWT payload     | ERR_INVALID_KB_JWT_IAT             |
 | sd_hash in KB JWT payload | ERR_INVALID_SD_HASH                |
 
+**cwt_vc Format VC Error Codes**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)